docs/testing.md

```diff
@@ -0,0 +1,72 @@
+## Run unit tests using CLI
+
+To run unit tests from the command line, use the following command from the BTP Manager main directory: 
+
+```shell
+make test
+```
+Details concerning `test` rule (prerequisites and recipe) are defined in `./Makefile`.
+
+By default, unit tests are run using `env-test` cluster. 
+Part of unit tests is implemented using [gingko](https://onsi.github.io/ginkgo/#top) library, but all unit tests are invoked using `go test ./... <some options>`
+You can find exact invocation reflected in the console output along with messages confirming whether `env-test` is used.
+```
+STEP: bootstrapping test environment @ 01/13/23 08:24:45.981
+2023-01-13T08:24:45.981+0100  DEBUG   controller-runtime.test-env     starting control plane
+```
+
+### Run unit tests on existing cluster
+
+You can run tests on an existing cluster (not the `env-test`) setting the value of the environment variable `USE_EXISTING_CLUSTER` to true.
+
+```shell
+USE_EXISTING_CLUSTER=true make test
+```
+
+> **NOTE:** Test suite assumes the proper state of the cluster before running. If problems with left-over resources occur, you can recreate the cluster or remove resources manually.
+
+### Test output verbosity
+
+Setting for the `go test` verbosity is `-v` (verbose, print the full output event for passing tests). This can be changed in `make` recipe. 
```

Comment:
```suggestion
The setting for the `go test` verbosity is `-v` (verbose, print the full output event for passing tests). This can be changed in the `make` recipe. 
```